Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml

```diff
@@ -0,0 +1,652 @@
+  "7":
+    id: "7"
+    type: title
+    task:
+      version: -1
+      name: Done
+      description: Done
+      type: title
+      iscommand: false
+      brand: ""
+      id: dca7b75c-681b-4ed9-a9b8-94198974c0db
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    taskid: dca7b75c-681b-4ed9-a9b8-94198974c0db
+  "8":
+    id: "8"
+    type: regular
+    task:
+      version: -1
+      name: Get sample
+      description: get sample by sample id
+      script: '|||vmray-get-sample'
+      type: regular
+      iscommand: true
+      brand: ""
+      id: e04293bd-c0a1-4f96-8abc-444ea5d42cee
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      sample_id:
+        complex:
+          root: VMRay.Submission
+          accessor: SampleID
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 377.5,
+          "y": 1070
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    taskid: e04293bd-c0a1-4f96-8abc-444ea5d42cee
+  "10":
+    id: "10"
+    type: regular
+    task:
+      version: -1
+      name: Get submission
+      description: Gets a submission result
+      script: '|||vmray-get-submission'
+      type: regular
+      iscommand: true
+      brand: ""
+      id: a7dcb1e4-dbfc-4e16-a586-f5ba0c9fbb7d
+    nexttasks:
+      '#none#':
+      - "1"
+    scriptarguments:
+      submission_id:
+        complex:
+          root: VMRay.Submission
+          accessor: SubmissionID
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 377.5,
+          "y": 720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    taskid: a7dcb1e4-dbfc-4e16-a586-f5ba0c9fbb7d
+  "11":
+    id: "11"
+    type: regular
+    task:
+      version: -1
+      name: Get analysis
+      description: Retrieves all analysis details for a specified sample.
+      script: '|||vmray-get-analysis-by-sample'
+      type: regular
+      iscommand: true
+      brand: ""
+      id: c3457fa4-7b07-41cb-b9e5-f0b523141429
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      limit: {}
+      sample_id:
+        complex:
+          root: VMRay
+          accessor: Sample.SampleID
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 377.5,
+          "y": 1245
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    taskid: c3457fa4-7b07-41cb-b9e5-f0b523141429
+  "12":
+    id: "12"
+    type: condition
+    task:
+      description: check term
+      version: -1
+      name: Is analysis suspicious or malicious?
```

Comment:
Please consider to organize the task for malicious and suspicious In a column and not in parallel so the path will be more clear to the user.

Reply:
Done in 2b657c1